rpc/documentation/api.md

```diff
@@ -1398,8 +1398,13 @@ transaction was seen.
   - `bool internal`: Whether the output pays to an address derived from the
     account's internal key series.  This often means the output is a change
     output.
+
   - `int64 amount`: The amount for this output.
-  - `string address`: The address that this output originated from.
+
+  - `string address`: The address to which the output script pays.  Note that 
+    this field may be null if the output script cannot parse a readable address.
```

Comment:
language still isn't quite right.. scripts don't parse, we parse them.  I would say:

> The address that the output paid to, if the output script can be parsed to a known address type.  Otherwise this will be null or the empty string.

Had to also add that last bit about the empty string since I think we can't actually make these null since strings aren't nullable in go.